prowler/providers/aws/services/kinesis/kinesis_service.py

```diff
@@ -0,0 +1,74 @@
+from enum import Enum
+
+from pydantic import BaseModel
+
+from prowler.lib.logger import logger
+from prowler.lib.scan_filters.scan_filters import is_resource_filtered
+from prowler.providers.aws.lib.service.service import AWSService
+
+
+class Kinesis(AWSService):
+    def __init__(self, provider):
+        # Call AWSService's __init__
+        super().__init__("kinesis", provider)
+        self.streams = {}
+        self.__threading_call__(self._list_streams)
+        self._describe_stream()
```

Comment:
If you are going to do it this way you should take advantage of the parallelism of the `__threading_call__` function since you have the listed streams.
```suggestion
        self.__threading_call__(self._describe_stream, streams.values())
```